modules/ROOT/pages/composite-databases/index.adoc

```diff
@@ -1,11 +1,88 @@
 [role=enterprise-edition]
 [[composite-databases]]
 = Composite databases
-:description: This section describes administration and operation of composite databases. 
+:description: An introduction to Composite databases.
 
-* xref:composite-databases/introduction.adoc[Introduction]
-* xref:composite-databases/queries.adoc[Queries]
-* xref:composite-databases/considerations.adoc[Further considerations]
-* xref:composite-databases/sharding-with-copy.adoc[Sharding data with the `copy` command]
+[[composite-databases-introduction]]
+== Introduction
 
+A Composite database is a special type of database introduced in Neo4j 5.
+It supersedes the implementation of the previous Fabric in Neo4j 4.
 
+Fabric as a concept describes how different parts of a domain (graph data and computing resources), which can be in the same or multiple Neo4j DBMSs, can work as a unified system with a single access point.
```

Comment:
This sentence is complicated - can it be simplified?

Reply:
Thank you so much, @tobias-johansson, for the review. Could you take a second look? I tried to explain it in a different way.